core/services/chainlink/testdata/config-full.toml

```diff
@@ -0,0 +1,286 @@
+Dev = true
+ExplorerURL = 'http://explorer.url'
+InsecureFastScrypt = true
+ReaperExpiration = '168h0m0s'
+Root = 'test/root/dir'
+ShutdownGracePeriod = '10s'
+FeatureFeedsManager = true
+FeatureUICSAKeys = true
+FeatureLogPoller = true
+FMDefaultTransactionQueueDepth = 100
+FMSimulateTransactions = true
+FeatureOffchainReporting2 = true
+FeatureOffchainReporting = true
+
+[Database]
+ListenerMaxReconnectDuration = '1m0s'
+ListenerMinReconnectInterval = '5m0s'
+Migrate = true
+ORMMaxIdleConns = 7
+ORMMaxOpenConns = 13
+TriggerFallbackDBPollInterval = '2m0s'
+
+[Database.Lock]
+Mode = 'advisory'
+AdvisoryCheckInterval = '5m0s'
+AdvisoryID = 345982730592843
+LeaseDuration = '1m0s'
+LeaseRefreshInterval = '1s'
+
+[Database.Backup]
+Dir = 'test/backup/dir'
+Frequency = '1h0m0s'
+Mode = 'full'
+OnVersionUpgrade = true
+URL = 'http://test.back.up/fake'
+
+[TelemetryIngress]
+UniConn = true
+Logging = true
+ServerPubKey = 'test-pub-key'
+URL = 'https://prom.test'
+BufferSize = 1234
```

Comment:
Yeah, I was actually thinking we could tighten up a lot of these 'buffer' fields with much smaller ints, since they would just OOM the node when they try to make a channel of that size anyways. In this case: `make(chan TelemPayload, tc.telemBufferSize)`

Reply:
SGTM